docs/neural_modeling/plot_01_current_injection.py

```diff
@@ -302,12 +303,9 @@
 # %%
 #
 # Now that we've done all this preparation, let's make a plot to more easily
-# visualize the data.
+# visualize the data. We're hiding the details of the plotting function for the purposes of this
+# tutorial, but you can find it in [the source code](https://github.com/flatironinstitute/nemos/blob/development/docs/neural_modeling/examples_utils/plotting.py) if you are interested.
```

Comment:
I want to do something else with this `examples_utils` code, so I've opened an issue about it #158

Reply:
I agree with @billbrod, it is annoying that people have to download extra code to run the notebooks